cypress.json

```diff
@@ -0,0 +1,7 @@
+{
+  "chromeWebSecurity": false,
```

Comment:
Hi @klubiniecki ,

Is there a reason why we've chosen to disable web security for chrome-based browsers ? 
Should we leave it at its default setting until we hit a problem in our testing?

I did a bit of digging around about the option, and found 2 relevant docs:

https://docs.cypress.io/guides/guides/web-security.html
https://docs.cypress.io/guides/references/configuration.html

So it seems like the option is for enabling/disabling same-origin policy and insecure mixed content during cypress' testing... 
I think we haven't hit any of those problems with our testing yet, right?

Reply:
You’re right, it proved useful down the road in my previous project. Happy to leave it out!